ghub.py

```diff
@@ -610,13 +610,47 @@ def create_issue(issue_text=None):
         print("Sorry, something bad happened: " + str(result))
 
 
+def close_issue(number, issue_text=None):
+    """
+    Closes an opened issue.
+
+    PATCH /repos/:owner/:repo/issues/:number
+
+    """
+    (upstream_user, upstream_repo) = get_user_and_repo('upstream', 'origin')
+    if issue_text:
+        data = json.dumps({'body': issue_text}).encode('utf8')
```

Comment:
Could this whole block be replaced with a call to `post_issue_comment(number, issue_text)`?

Reply:
@ccstolley this block in fact could be replaced with `post_issue_comment(number, issue_text)`. 

My concern here is what kind of functionality would make sense out of these 2 options I have in mind?

1. if commenting fails, do we want to abort the whole closing of the issue? If this is a YES, I will need to either leave this code block in there or make a change in `post_issue_comment(number, issue_text)` function to abort if commenting fails. 

2. If we don't care about aborting closing an issue due to a comment failure than I say yes, method `post_issue_comment(number, issue_text)` could be used here.

Reply:
Could we modify `post_issue_comment()` to just return a boolean to indicate success? That way you could abort closing the issue if it failed.

Reply:
That should work!